src/schedule/zephyr_ll.c

```diff
@@ -138,8 +138,18 @@ static inline enum task_state do_task_run(struct task *task)
 	perf_cnt_init(&task->pcd);
 #endif
 
+#ifdef CONFIG_SOF_TELEMETRY
+	const uint32_t begin_stamp = (uint32_t)telemetry_timestamp();
+#endif
+
 	state = task_run(task);
 
+#ifdef CONFIG_SOF_TELEMETRY
+	const uint32_t current_stamp = (uint32_t)telemetry_timestamp();
+
+	telemetry_update(begin_stamp, current_stamp);
+#endif
+
```

Comment:
So this is called more often after the patch, I guess this is intended in the patch.

Reply:
Hm, I intended this to be more aligned with  performance overlay... but this does seem to update the values more often which seems unnecessary. Not sure how it should work. I could delete this one commit for now, as this not a main focus of this PR.